local-volume/provisioner/pkg/deleter/deleter.go

```diff
@@ -81,10 +81,10 @@ func (d *Deleter) deletePV(pv *v1.PersistentVolume) error {
 	}
 
 	// TODO: Get volType from PV.
-	volType := common.VolumeTypeFile
+	volType := v1.PersistentVolumeFilesystem
 	isblk, _ := d.VolUtil.IsBlock(mountPath)
```

Comment:
I think I would determine the cleanup method only based on the PV volume mode.  Because an app could format a filesystem on top of their block device.

Reply:
Fixed.